### ISSUE TITLE
Windows environment variables in construct.yaml

### ISSUE THREAD

**Issue by scottwsides** [conda/constructor#527]
Sorry, I know there's a closed request about this but it would really be helpful. I'd like to specify windows environment variables (eg %HOMEPATH% for the default user install location but everything I try fails. Is there a way to do this?

**Comment by jaimergp** [comment]
The environment variables you set at build time would be rendered with the value you have at build time (so HOMEPATH would be set to the build machine HOMEPATH, potentially different to the machine of the user installing your software). I _think_ you can still use `%HOMEPATH%` directly in the YAML, since NSIS will inherit the environment from the user and render it. After all, it's what the NSIS template is doing:

https://github.com/conda/constructor/blob/a372ee97abdf852bf37156569a3b94033c2c90cd/constructor/winexe.py#L66

So, in a nutshell, you can just use it directly.
